setup/apache.yml

```diff
@@ -18,7 +18,7 @@ apache_cosign_deny_friend:     "global=app_cosign_deny_friend default=no"
 
 apache_static_path:
   global: deploy_root
-  output: File.join(deploy_root, app_name, "shared", "public")
+  output: File.join(deploy_root, app_name, "current", "public")
```

Comment:
The symlink, `current`, won't exist when a Capistrano deploy fails. Which is probably not a problem, since the application won't be running anyway.  This would prevent apache from serving derivatives being served out when the application isn't up?

How was `apache_static_path` affecting the config? The xsendfile appears to use `apache_derivatives_path`?